### PR TITLE
Ft project details

### PR DESCRIPTION
**Pull request by DjibrilM**
# In the following pull request i added 

- project detail modal

### CODE REVIEW

File: assets/styles/details.css [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+  }
+
+  .project-desc .parag {
+    flex-basis: 70%;
+  }
+
+  .modal {
+    max-width: 900px;
+  }
+}
```
Comment (brainconnect93) [inline assets/styles/details.css:150]: - [ ] Great effort working on your pop up modal project, Currently i notice that the pop up card is hitting the wall of the page at the top and bottom which makes the `see live` and `see source` button get cut out, i would kindly suggest you work on it and be sure to have a design that looks like what we have in the Figma design

| Figma Design 👇 | Your Design 👇 |
| ------------- | ------------- |
| real-pop |Nj |

File: index.html [hunk 2]
```diff
@@ -8,15 +8,92 @@
...
+                        <svg width="24" height="24" class="close-btn" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                            <path fill-rule="evenodd" clip-rule="evenodd"
+                                d="M5.29289 5.29289C5.68342 4.90237 6.31658 4.90237 6.70711 5.29289L12 10.5858L17.2929 5.29289C17.6834 4.90237 18.3166 4.90237 18.7071 5.29289C19.0976 5.68342 19.0976 6.31658 18.7071 6.70711L13.4142 12L18.7071 17.2929C19.0976 17.6834 19.0976 18.3166 18.7071 18.7071C18.3166 19.0976 17.6834 19.0976 17.2929 18.7071L12 13.4142L6.70711 18.7071C6.31658 19.0976 5.68342 19.0976 5.29289 18.7071C4.90237 18.3166 4.90237 17.6834 5.29289 17.2929L10.5858 12L5.29289 6.70711C4.90237 6.31658 4.90237 5.68342 5.29289 5.29289Z"
+                                fill="#67798E" />
+                        </svg>
+
+                    </div>
+
+                    <ul>
+                        <li>CANOPY</li>
```
Comment (brainconnect93) [inline index.html:32]: - [ ] Hello @DjibrilM, As requested in this milestone, the pop up modal card should be created dynamically using `array of an object` in your JavaScript file and all the hardcoded line of codes in your HTML files should be removed

File: index.html [hunk 3]
```diff
@@ -261,7 +338,7 @@ <h2 class="project-title">Multi-Post <br> Stories</h2>
                 </div>
 
                 <div class="project-button-link-container">
-                    <a href="https://github.com/DjibrilM/J-snippet/tree/main">
+                    <a href="#">
                         See project
                     </a>
                 </div>
```
Comment (brainconnect93) [inline index.html:344]: - [ ] Hello colleague, I could not highlight all the line of codes needed for this comment, but I would kindly advise you stick to the project requirement as much as possible, The goal of this milestone, is to have all your project cards that has been hardly coded in your HTML file be created dynamically in your JavaScript file using `array of an object`, Then all the line of code in your HTML file will be removed.

Below is the requirement 👇 

image

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @DjibrilM, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 

- Descriptive Pull Request title and summary. ✔️
- User clicks the button, the popup with details about the project appears ✔️
- User clicks the close (X) button, the popup disappears ✔️
- Modal popup data is populated with the data from the projects array ✔️
- Good use of Github flow ✔️

### Changes Required

- [ ] I would kindly suggest you work on the MIT license link of your project in the README file. Currently, the link is broken and throwing error 404. Basically i notice your project is missing the License file in the root directory kindly add it.

MIT 👇 

mit


- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :
Hi @DjibrilM , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights :tophat: :+1: 
- No Linter Errors :100: 
- Meaningful Commit Messages :+1: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions :ringed_planet: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AdnanOlarmmi — APPROVED** [review]
Hi @DjibrilM,

This is Adnan Is-haq, your beloved code reviewer, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved office man

## Highlights
- Fixed changes.

## Optional suggestions
_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**